libraroo-backend/src/physical-books/physical-books.service.ts

```diff
@@ -0,0 +1,4 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class PhysicalBooksService {}
```

Comment:
![image](https://github.com/user-attachments/assets/6613eb0c-c0f6-45ca-bc92-802be1e40a25)

Reply:
We didnt discuss specifically what we need